# current memory limit for session is 32KB which small. It makes proftpd
# session to fail as soon as client connects. bumping it to 32MB improves
# things a bit. We fail at runtime linker relocation when modules are
# loaded. 64MB make proftpd process happy.
#
# rhe patch has not been submitted to upstream.

--- a/tests/t/lib/ProFTPD/Tests/Modules/mod_rlimit.pm
+++ b/tests/t/lib/ProFTPD/Tests/Modules/mod_rlimit.pm
@@ -36,7 +36,7 @@ sub rlimit_memory {
   my $tmpdir = $self->{tmpdir};
   my $setup = test_setup($tmpdir, 'rlimit');
 
-  my $rlimit_mem = '32K';
+  my $rlimit_mem = '64M';
 
   my $config = {
     PidFile => $setup->{pid_file},
